src/main/kotlin/io/sip3/salto/ce/util/SIPMessageUtil.kt

```diff
@@ -79,8 +82,34 @@ fun SIPMessage.headersMap(): Map<String, String> {
     }
 }
 
+fun SIPMessage.hasSdp(): Boolean {
```

Comment:
Implementation below is incorrect.
We have to check if `SIPMessage` has content and also check if this content matches `sdp`